index.html

```diff
@@ -1,14 +1,100 @@
+            <a href="learn-more.html"><br><button id="more">Learn More</button></a><hr>
+            <h4>Something of Less Note</h4>
+            <img src="https://i.imgur.com/SMwTgQ5.png" alt="Black Canyon of the Gunnison" class="side";/>
+            Quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda omnis, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates 
+            <a href="learn-more.html"><br><button id="more">Learn More</button></a>  
+        </div>
+
+   </div>
+   <!--I made the footer and reused it on each page, creating a grid within a grid for each instance. -->
+<div id="footer">
+    <section id="questions">Questions or comments? Get in Touch:</section>
+    <section id="form"> 
+      <input type="text" placeholder="Name" id="input1"></input>
+      <input type="text" placeholder="Email" id="input2"></input>
+      <textarea type="text" placeholder="Message">Message</textarea>
+      <br><button id="more">Send Message</button>
+    </section>
+    <container id="info">
+      <div class="writing">iendis distinctio quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda omnis</div>
+      <div class="data1">
+        <img src="https://www.flaticon.com/premium-icon/icons/svg/2932/2932143.svg" class="icon"/> 1234 Somehwere Road<br>     Burlington, VT 00000<br>USA<br><br>
+        <img src="https://www.flaticon.com/premium-icon/icons/svg/3059/3059446.svg" class="icon"/><a href="tel:1(888)867-5309" class="link">888-867-5309</a><br><br>
+        <img src="https://www.flaticon.com/premium-icon/icons/svg/542/542638.svg" class="icon"/><a href="mailto:info@untitled.come" class="link">info@untitled.com</a></div>
+      <div class="data2">
+        <img src="https://www.flaticon.com/svg/vstatic/svg/2111/2111738.svg?token=exp=1613100350~hmac=e3aa8c5bbf7b2e3beb6279c6e4277904" class="icon"/><a href="www.twitter.com" class="link">@untitled</a><br><br>
+        <img src="https://www.flaticon.com/svg/vstatic/svg/1384/1384031.svg?token=exp=1613100414~hmac=4719772d53787383996fd624606e8f52" class="icon"/><a href="www.instagram.com" class="link">instagram.com</a><br><br>
+        <img src ="https://www.flaticon.com/svg/vstatic/svg/2301/2301129.svg?token=exp=1613099848~hmac=6206a5262339999674976f18ece87fa7" class="icon"/><a href="www.dribble.com" class="link">dribble.com</a><br><br>
+        <img src="https://www.flaticon.com/svg/vstatic/svg/1384/1384005.svg?token=exp=1613100476~hmac=b4ce5f91f1434285ea07db0bba2c93f7" class="icon"/><a href="www.facebook.com" class="link">facebook.com</a></div>
+  </container>
+      <div id="footer1">to delete</div>
+      <div id="footer2">to delete</div>

```

Comment:
This is another spot that we could utilize the `grid-template-area` and set these sections to a "." to say that this grid space is empty